packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/CreateRulesTests.java

```diff
@@ -188,7 +189,7 @@ public void requireReadPermissionToCreateRule()
         restClient.assertLastError().containsSummary("Insufficient permissions to manage rules");
     }
 
-    /** Check that a Collaborator cannot create a rule in a private folder. */
+    /** Check that a Collaborator cannot create a rule in a folder in a private site. */
     public void siteCollaboratorCannotCreateRule()
```

Comment:
Do we need `@Test (groups = { TestGroup.REST_API, TestGroup.RULES })` annotation for the method to be executed?